bindings/nodejs/npm/darwin-arm64/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@glaredb/glaredb-darwin-arm64",
+  "version": "0.0.1",
```

Comment:
Will these need to be manually updated on version bumps?

Reply:
Yeah, unfortunately the js tooling isnt quite as nice as `maturin`. So we'll have to manually bump this when releasing.

Reply:
If Grey's good with this, then this all seems fine to me. We can look at possibly writing a script for this at some point.

Reply:
Being able to script the version shouldn't be too hard. 👍🏻 good with the PR to get things rolling.